lib/travis/api/app/endpoint/authorization.rb

```diff
@@ -288,9 +287,10 @@ def fetch
               if user
                 rename_repos_owner(user.login, info['login'])
                 user.update_attributes info
+                Travis.run_service(:sync_user, user) if user.previous_changes[:github_oauth_token]
               else
                 self.user = ::User.create! info
```

Comment:
@renee-travisci hah. you're right. not sure what i was looking at. i've removed that.